src/App.tsx

```diff
@@ -15,48 +15,102 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  reverse: boolean,
+  type: number,
+};
+
+function ReorderGoods(goods: string[], { reverse, type }: ReorderOptions) {
```

Comment:
```suggestion
function getReorderGoods(goods: string[], { reverse, type }: ReorderOptions) {
```
Let's name functions according to these rules:
![image](https://user-images.githubusercontent.com/96409563/236399161-06b61921-134f-4522-8da6-062f03b2ab95.png)